evaluator/evaluator.ml

```diff
@@ -53,6 +53,34 @@ end
 
 let setup () = JsooTop.initialize ()
 
+let insertModule name content = 
+  begin 
+    let result = try  
+      let name = Js.to_string name in
+      let content = Js.to_string content in
+      let file_name = ("/static/" ^ (String.capitalize_ascii name) ^ ".re") in 
+      let _ = File__System.createOrUpdateFile file_name content in
+      Execute.mod_use_file file_name
+    with 
+      | exn -> 
+        let buffer = Buffer.create 100 in
```

Comment:
This is great, thank you for investigating! @thangngoc89

Reply:
My pleasure!